hail/src/main/scala/is/hail/services/shuffler/ShuffleClient.scala

```diff
@@ -18,26 +18,7 @@ import org.apache.log4j.Logger
 object ShuffleClient {
   private[this] val log = Logger.getLogger(getClass.getName())
 
-  lazy val sslContext = is.hail.services.shuffler.sslContext(
-    getClass.getResourceAsStream("/non-secret-key-and-trust-stores/client-keystore.p12"),
-    "hailhail",
```

Comment:
Did you not need this anymore?

Reply:
nope! I also removed those files. Everything uses the official Hail SSL/TLS infrastructure now. That happens in `getSSLContext.getSocketFactory()` in `DeployConfig.socket`.